app/services/set_decline_by_default.rb

```diff
@@ -0,0 +1,35 @@
+class SetDeclineByDefault
+  def initialize(application_form:)
+    @application_form = application_form
+  end
+
+  def call
+    return false if pending_application_choices.any? || application_choices.none?(&:offered_at)
```

Comment:
As already discussed f2f, there are two checks that the service needs to make for this bit of work:
1. whether there are still application choices waiting for a provider decision (determines if DBD is set)
2. whether an application choice is an offer (DBD should be set only on 'offer' choices)

I have decided to keep the timestamps (```offered_at```, ```rejected_at```) but switch to ```status == 'awaiting_provider_decision'``` for check 1 and ```status == 'offer'``` for check 2 because this is better aligned with our state machine workflow.

This is also more consistent with the new logic of setting DBD only on 'offer' choices.